storage/src/main/java/tech/pegasys/teku/storage/server/network/DatabaseNetwork.java

```diff
@@ -39,12 +40,24 @@ public class DatabaseNetwork {
   @VisibleForTesting
   final String depositContract;
 
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  @JsonProperty("deposit_chainId")
+  @VisibleForTesting
+  final Long depositChainId;
+
   @JsonCreator
   DatabaseNetwork(
       @JsonProperty("fork_version") final String forkVersion,
-      @JsonProperty("deposit_contract") final String depositContract) {
+      @JsonProperty("deposit_contract") final String depositContract,
+      @JsonProperty("deposit_chainId") final Long depositChainId) {
```

Comment:
the name here is a mix of 2 worlds, i think deposit_chain_id would be more consistent with other fields.